src/config.js

```diff
@@ -108,6 +108,17 @@ let loadUserConfig = function (configDir: string, configOptions: ConfigOptions)
   return config
 }
 
+function shell (onWindows: boolean = false): string {
+  let shellPath
+  if (onWindows) {
+    shellPath = (process.env['SHELL'] && process.env['SHELL'].split(`/`)) || (process.env['COMSPEC'] && process.env['COMSPEC'].split(/\\|\//))
+  } else {
+    shellPath = process.env['SHELL'] && process.env['SHELL'].split(`/`)
+  }
```

Comment:
@RasPhilCo could you simplify this so that the SHELL part is not duplicated?
```
if (SHELL) {
  shellPath = SHELL.split
} else if (onWindows && COMSPEC) {
  shellPath = COMSPEC.split
} else {
  shellPath = ['unknown']
}
```